docs/tests/integration/test_remoted/test_active_response/test_active_response.md

```diff
@@ -0,0 +1,24 @@
+# Test active response
+
+## Overview
+These tests will check if active response command is sent correctly to the agent.
+
+## Objective
+
+The objective is to check that the manager correctly sent active response to the agent and that this one receives it.
```

Comment:
I would add more detail here about the test. In fact, the format of the active response messages, how they must appear in the agent, etc.